### PR TITLE
Don't show a message bubble around polls

### PR DESCRIPTION
**Pull request by t3chguy**
Fixes https://github.com/vector-im/element-web/issues/19953

image

---
This PR currently has no changelog labels, so will not be included in changelogs.

Add one of: `T-Deprecation`, `T-Enhancement`, `T-Defect`, `T-Task` to indicate what type of change this is plus `X-Breaking-Change` if it's a breaking change.

Preview: https://61b9f3f54937e114346fcb62--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: res/css/views/rooms/_EventBubbleTile.scss [hunk 1]
```diff
@@ -312,6 +312,10 @@ limitations under the License.
     }
 }
 
+.mx_EventTile.mx_EventTile_noBubble[data-layout=bubble] {
+    --backgroundColor: tranparent;
```
Comment (andybalaam) [inline res/css/views/rooms/_EventBubbleTile.scss:316]: typo: missing "s"

**Review by andybalaam — APPROVED** [review]
After the one comment, looks good!

File: res/css/views/rooms/_EventBubbleTile.scss (hunk 1 above)
Comment (t3chguy) [reply res/css/views/rooms/_EventBubbleTile.scss:316]: well that explains why it didn't syntax highlight, hilariously enough it works regardless :D

Comment (t3chguy) [reply res/css/views/rooms/_EventBubbleTile.scss:316]:
```suggestion
    --backgroundColor: transparent;
```

**Review by daniellekirkwood — APPROVED** [review]
This looks great, thanks!

**Review by amshakal — APPROVED** [review]

**Pull request merged**